doc/misc/npm-config.md

```diff
@@ -385,6 +385,14 @@ A proxy to use for outgoing https requests. If the `HTTPS_PROXY` or
 `https_proxy` or `HTTP_PROXY` or `http_proxy` environment variables are set,
 proxy settings will be honored by the underlying `request` library.
 
+### if-present
+
+* Default: false
+* Type: Boolean
+
+If true, npm does not exit with non-zero when `run-script` is used for a script
```

Comment:
I'd like to see this more explicitly describe why you might want to use this option.